content/news/2022-05-27-release-22.05-highlights.md

```diff
@@ -0,0 +1,119 @@
++++
+title = "Release 22.05 Highlights"
+date = 2022-05-27T02:40:00Z
+type = "post"
+description = "Highlights of the 22.05 release."
+in_search_index = true
++++
+
+Helix is a modal text editor with built-in support for multiple selections,
+Language Server Protocol (LSP), tree-sitter, and experimental support for Debug
+Adapter Protocol (DAP).
+
+Today marks the 22.05 release, a featureful release with over double the number
+of contributors as last release. A big thanks to all involved! 🙌
+
+Let's check out the flashy features of 22.05.
+
+## Indentation Rework
+
+The indentation system has been fully reworked. Indentation still works by
+querying the parsed tree-sitter syntax tree for a document, but now query files
+can use the full expressive power of tree-sitter queries rather than just
+simple node names. Expect more robust indentation with fewer edge cases.
+
+## Configurable Gutters
+
+Gutters can now be configured in your `config.toml` file. For example, you can
+remove the line-number gutter while still showing LSP diagnostics.
+
+```toml
+[editor]
+gutters = ["diagnostics"] # default is ["diagnostics", "line-numbers"]
+```
+
+## Local Language Configuration
+
+Language configuration can now be specified local to a project. Create a
+`.helix` directory and a `languages.toml` under it to override values from the
+[default language configuration]. For example, override a C project's indents
+to use tabs instead of spaces for indentation:
+
+```toml
+# my_c_project/.helix/languages.toml
+[[language]]
+name = "c"
+indent = { tab-width = 8, unit = "\t" }
+```
+
+## Rulers
+
+{{ asciinema(id="rulers", width="94", height="25") }}
+
+Vertical rulers may now be configured through the `rulers` option. Use them
+as a guide when adhering to a maximum line length.
+
+## Show Visible Whitespace
+
+{{ asciinema(id="visible-whitespace", width="94", height="25") }}
+
+Whitespace characters like spaces, tabs, and newlines may now be rendered. Use
+the `whitespace.render` option either at runtime with the `:set` command or
+in your `config.toml` under the editor section. The characters used for tabs,
+spaces, non-breaking space, and newlines may also be customized as well as
+whether any of the characters are rendered at all. In fact, for the remaining
+asciicasts in this post, you'll be seeing my custom tab and newline characters!
```

Comment:
Can we add a note here that visible whitespace can make working with helix's selection model more intuitive, at least in the case of line endings? I think this could be a big boon to new users.